File: LeetCode/LeetCode Practice Problems/268- Missing Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int expected = n * (n + 1)/2;
        int actual = 0;
        
        for(auto i : nums) {
            actual += i;
        }
        
        return expected - actual;
        
    }
};

File: LeetCode/LeetCode Practice Problems/268-Missing Number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int target = n * (n + 1)/2;
        int cur = 0;
        
        for(int num: nums) {
            cur += num;
        }
        return target - cur;
    }
}
